include/lin/core/operations/mapping_transpose.hpp

```diff
@@ -0,0 +1,107 @@
+// vim: set tabstop=2:softtabstop=2:shiftwidth=2:expandtab
```

Comment:
? is this supposed to be  there

Reply:
This is just so if you open the file in vim it sets tab width to 2 spaces and expands tabs.